### PR TITLE
adding new astra files for dr19 and ipl3

### PR DESCRIPTION
**Pull request by ajwheeler**
I've added the astra files (I'll do apogee DRP/apoee redux in future PR).  I've also updated the ipl3.cfg to include the 2 new files since it was updated last.

### CODE REVIEW

**Comment by havok2063** [comment]
@ajwheeler can you please merge in the latest main?

**Comment by havok2063** [comment]
The test is failing because there is a syntax error in the key name.  Can you also give this a more representative title of the work in the PR?

**Comment by ajwheeler** [comment]
This branch include stuff from Sean and Joel, not just what I wrote.  (This is why I was asking about whether I should be pushing commits to it.)

**Comment by ajwheeler** [comment]
And sorry to be thick, but I don't know what syntax error you are referring to.

**Comment by joelbrownstein** [comment]
Adam, In hindsight it would have been simpler to create a fresh branch for your PR.

**Comment by ajwheeler** [comment]
Happy to `git revert` the changes and put them on a separate branch.  That's actually what I did at first, but when I realized there was already a dr19 branch, I assumed that that's where dr19 changes should go.  I think using a separate branch is likely to require a manual merge later (two parallel `dr19.cfg`s), but that's obviously not a huge problem.

**Comment by joelbrownstein** [comment]
I personally prefer a fresh branch, but only if you want to!

**Comment by ajwheeler** [comment]
If it's up to me I'd rather merge this one and avoid the conflict in the future, but it's not a strong opinion.

**Comment by joelbrownstein** [comment]
That's fine, since I don't think any of the old changes I made are problematic (just cleanup)

However, the syntax error appears related to the name astraAllStarASCAP.  I don't see anything obvious -- Brian do you have any insight into this?

File: tests/test_tree.py [hunk 1]
```diff
@@ -302,4 +302,4 @@ def test_ipl3_config():
     tree = Tree(config='ipl3')
     assert 'MWM_ASTRA' in tree.environ['MWM']
     assert 'ipl-3/spectro/astra' in tree.environ['MWM']['MWM_ASTRA']
-    assert tree.paths['astraAllStarBest'] == '$MWM_ASTRA/{v_astra}/summary/astraAllStarBest-{v_astra}.fits'
\ No newline at end of file
+    assert tree.paths['astraAllStarASCAP'] == '$MWM_ASTRA/{v_astra}/summary/astraAllStarASCAP-{v_astra}.fits'
```
Comment (havok2063) [inline tests/test_tree.py:305]: This is the line that I think is causing the test to fail.

Comment (joelbrownstein) [reply tests/test_tree.py:305]: Its just a typo

Comment (joelbrownstein) [reply tests/test_tree.py]: I fixed the typo in the ipl3 test and commited.

**Comment by joelbrownstein** [comment]
I checked on the changes prior to Adam's commits, and I think they're not concerning.

Brian, please proceed with the merge if you're happy with this PR as it is.

File: tests/test_tree.py (hunk 1 above)
Comment (ajwheeler) [reply tests/test_tree.py]: thanks

**Review by havok2063 — APPROVED** [review]

**Pull request merged**